opencti-platform/opencti-graphql/src/resolvers/file.js

```diff
@@ -28,7 +28,7 @@ const fileResolvers = {
     creator: (metadata, _, context) => creatorLoader.load(metadata.creator_id, context, context.user),
   },
   Mutation: {
-    uploadImport: (_, { file }, context) => uploadImport(context, context.user, file),
+    uploadImport: (_, args, context) => uploadImport(context, context.user, args),
```

Comment:
Have you checked that this code still works fine ?
https://github.com/OpenCTI-Platform/client-python/blob/65794bb194e9e87411d735f630308cdbeff4332b/pycti/api/opencti_api_client.py#L626-L634

Reply:
Can't hurt to check, but API is unchanged (we still expect an object args).

Reply:
However, we might want to update the python client to provide the new parameter `file_markings` if we want to align with the gql API.

Reply:
I'll do it in another PR, let's move on with this one